ExpenseTracker/ExpenseTracker/Views/Shared/_Layout.cshtml

```diff
@@ -32,7 +32,7 @@
     <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
 </head>
 <body class="e-bigger">
-    <partial name="_Sidebar" />
+    <partial name="_Sidebar" model="selectedWallet" />
```

Comment:
instead of passing **selectedWallet**, please use **CurrentUserService** to get current user's username.